docs/release-notes/NuGet-6.14.md

```diff
@@ -2,27 +2,130 @@
 title: NuGet 6.14 Release Notes
 description: Release notes for NuGet 6.14 including new features, bug fixes, and DCRs.
 author: zivkan
+ms.date: 5/6/2025
 ms.topic: conceptual
 ---
-# NuGet 6.14 Release Notes
 
-<!--
-    This is intentionally not yet added to TOC.md.
-    When this version is ready for release:
-    1. Rename this file to the next version
-    2. Change the version strings in this file to the next version
-    3. Use the release tool to create the real release notes for the version ready for release
-    4. Add the release notes to TOC.yml
--->
+# NuGet 6.14 Release Notes
 
 NuGet distribution vehicles:
 
 | NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
 |:---|:---|:---|
-| [**6.14**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.14](https://visualstudio.microsoft.com/downloads/) | [9.0.2xx](https://dotnet.microsoft.com/download/dotnet/9.0)<sup>1</sup> |
+| [**6.14.0**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.14.1](https://visualstudio.microsoft.com/downloads/) | [9.0.300](https://dotnet.microsoft.com/download/dotnet/9.0)<sup>1</sup> |
```

Comment:
why 17.14.1?

Reply:
I was trying to figure out if VS starts off with `.0` or .1 releases, and the 17.13 release notes start with .1: https://learn.microsoft.com/en-us/visualstudio/releases/2022/release-notes#17.13.1

Is there actually a .0 release?

Reply:
Starts with 0 for sure.